microsoft-edge/devtools-guide-chromium/dom/index.md

```diff
@@ -243,29 +244,29 @@ Press `Delete` to delete a node, as follows:
 <!-- ====================================================================== -->
 ## Access nodes in the Console
 
-DevTools provides a few shortcuts for accessing DOM nodes from the Console, or getting JavaScript references to each one.
+DevTools provides a few shortcuts for accessing DOM nodes from the **Console** tool, or getting JavaScript references to each one.
 
-### Reference the currently-selected node with $0
+#### Reference the currently-selected node with $0
 
 When you inspect a node, the `== $0` text next to the node means that you can reference this node in the Console with the variable `$0`.
 
 1. Open the [DOM Examples](https://microsoftedge.github.io/Demos/devtools-dom-get-started/) demo page in a new window or tab.
 
 1. In the rendered webpage, under **Reference the currently-selected node with $0**, right-click **The Left Hand of Darkness** and then select **Inspect**.
 
-1. Press the **Esc** key to open the Console Drawer.
+1. Press the **Esc** key to open the **Console** tool in the Quick View panel.
 
 1. Type `$0` and press **Enter**.  The result of the expression shows that `$0` evaluates to `<p>The Left Hand of Darkness</p>`:
 
-   ![The result of the first $0 expression in the Console](./index-images/dom-elements-highlighted-reference-currently-selected-node-console-1.png)
+   ![The result of the first $0 expression in the Console](./index-images/reference-currently-selected-node-console-1.png)
 
 1. Hover on the result.  The node is highlighted in the viewport.
```

Comment:
Global: use "hover over" (like older version of doc), not "hover on".
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/mouse-mouse-interaction-terms
```suggestion
1. Hover over the result.  The node is highlighted in the viewport.
```